src/main/java/mate/jdbc/Main.java

```diff
@@ -1,7 +1,23 @@
 package mate.jdbc;
 
+import mate.jdbc.dao.ManufacturerDao;
+import mate.jdbc.lib.Injector;
+import mate.jdbc.model.Manufacturer;
+
 public class Main {
+    private static final Injector injector = Injector.getInstance("mate.jdbc");
+
     public static void main(String[] args) {
+        ManufacturerDao manufacturerDao
+                = (ManufacturerDao) injector.getInstance(ManufacturerDao.class);
+        Manufacturer bmw = new Manufacturer("BMW", "Germany");
+        Manufacturer audi = new Manufacturer("Audi","Germany");
+        manufacturerDao.create(bmw);
+        manufacturerDao.delete(1L);
```

Comment:
```suggestion
        Manufacturer createdBmw = manufacturerDao.create(bmw);
        manufacturerDao.delete(createdBmw.getId());
```
let's use `createdBmw.getId()` instead of `1L`
for example, if I run this code on my laptop it will not work. I don't have manufacturer with such id.